AdyenActions/Components/3DS2/Action handlers/3DS2+Delegated Authentication/ThreeDS2PlusDACoreActionHandler.swift

```diff
@@ -148,6 +148,108 @@
             }
             return nil
         }
+    
+        // MARK: - Delegated Authentication
+
+        /// This method checks;
+        /// 1. if DA has been registered on the device
+        /// 2. shows an approval screen if it has been registered
+        /// else calls the completion with a failure.
+        private func performDelegatedAuthentication(_ fingerprintToken: ThreeDS2Component.FingerprintToken,
+                                                    completion: @escaping (Result<String, DelegateAuthenticationError>) -> Void) {
+            let failureHandler = { completion(.failure(DelegateAuthenticationError.authenticationFailed(cause: nil))) }
```

Comment:
this closure should take an optional error as a parameter as a `cause` error

Reply:
Is there any way to provide `cause` ? This could be insightful for consumers..

Reply:
Done. Thanks ✅ Yes, made changes to propagate the error upstream.